src/web/controllers/admin/index_admin_controller.py

```diff
@@ -172,7 +172,7 @@ def _admin_render(
             'home': {
                 'title': _('Home'),
                 'template': 'index/home_tab.html',
-                'icon_class': 'bi-qr-code',
+                'icon_class': 'bi-house-fill',
```

Comment:
I remember Tim did not want to have the house icon to goto the home page because having the icon on a page means you already are on the home page.
```suggestion
                'icon_class': 'bi-qr-code',
```